.github/workflows/blank.yml

```
@@ -0,0 +1,36 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
```

```suggestion
name: build
```

rename file to `build.yaml`